### PR TITLE
Updated Noise Footprints and Building Noise

### PR DESCRIPTION
**Pull request by mclarke2**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_noise_evaluation_locations.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    Inputs:  
+        settings.urban_canyon_microphone_locations          - array of microphone locations on a building(s)  [meters] 
+        segment.conditions.frames.inertial.position_vector  - position of aircraft                            [boolean]                                     
+
+    Outputs: 
+    BM_THETA   - angle measured from building microphone in the x-z plane from microphone to aircraft 
+    BM_PHI     - angle measured from building microphone in the y-z plane from microphone to aircraft 
+    BML        - building microhpone locations
+    num_b_mic  - number of building microhpones
```
Comment (planes) [inline trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_noise_evaluation_locations.py:75]: update these

File: trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/generate_microphone_points.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    """   
+    
+    num_buildings            = len(building_locations) 
+    num_mics_on_xz_surface   = N_x*N_z 
+    num_mics_on_yz_surface   = N_y*N_z 
+    num_mics_on_xy_surface   = N_x*N_y
+    num_mics_per_building    = 2*(num_mics_on_xz_surface +num_mics_on_yz_surface) +  num_mics_on_xy_surface
+    b_mic_locations          = np.empty((num_mics_per_building*num_buildings,3))
+    
+    for i in range(num_buildings):
```
Comment (planes) [inline trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/generate_microphone_points.py:83]: This is vectorize-able

Comment (mclarke2) [reply trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/generate_microphone_points.py:83]: vectorized

File: trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_noise_evaluation_locations.py (hunk 1 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_noise_evaluation_locations.py:75]: Updated

**Review by planes — APPROVED** [review]

**Pull request merged**